library/psa_crypto.c

```diff
@@ -152,9 +153,13 @@ psa_status_t mbedtls_to_psa_error(int ret)
         case 0:
             return PSA_SUCCESS;
 
+#if defined(PSA_WANT_KEY_TYPE_AES)
```

Comment:
Weirdly this `switch` doesn't seem to cover `MBEDTLS_ERR_AES_BAD_INPUT_DATA` at all